docs/DeveloperGuide.md

```diff
@@ -633,13 +633,49 @@ Similar to [<ins>UC09](#use-case-uc09-add-phone-number-to-contact) except duplic
 * 2a.The tag list is empty.
   Use case ends.
 
-* 3a. The provided tag name does not exist.
-    * 3a1. The system displays an error message.
+* 3a. The contact already has the specified tag assigned to them.
+    * 3a1. System displays an error message.
+  
+      Use case ends.
+
+* 3b. User specified to force the tagging of a contact.
```

Comment:
I think for 3b and 3c might need to rephrase to:
3b. User specifies to force the tagging of a contact when the tag does not exist.
3b. User specifies to tag the contact without forcing it.

? Something like this to make the distinction a bit clearer